setup.py

```diff
@@ -35,12 +36,11 @@ def read(fname):
 
 
 def in_scriptdir(path):
-    return os.path.join(SCRIPTDIR, os.path.normpath(path))
+    return join(SCRIPTDIR, normpath(path))
 
 
 def rebuild_catalogs():
-    import subprocess
-    subprocess.call('python scripts/cmpmsgs.py librarian/locales', shell=True)
+    call('python scripts/cmpmsgs.py librarian/locales', shell=True)
```

Comment:
This is triggered right before `sdist`, among other things. I think it's not a bad thing to have, as tox will execute a `sdist` before running tests.